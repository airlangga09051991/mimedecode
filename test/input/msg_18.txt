From test Sat Feb  1 00:00:00 2014
Content-Type: text/plain; charset="us-ascii"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Foobar-Spoink-Defrobnit: wasnipoop; giraffes="very-long-necked-animals";
	spooge="yummy"; hippos="gargantuan"; marshmallows="gooey"

