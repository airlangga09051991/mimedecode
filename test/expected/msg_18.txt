From test Sat Feb  1 00:00:00 2014
Content-Transfer-Encoding: 7bit
X-Foobar-Spoink-Defrobnit: wasnipoop; giraffes="very-long-necked-animals";
	spooge="yummy"; hippos="gargantuan"; marshmallows="gooey"
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
X-MIME-Autoconverted: from us-ascii to utf-8 by test id mimedecode

